src/Utility/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_library(utility STATIC
```

Comment:
Nice use of cmakelists library convention.

Reply:
<3